core/create/CHANGELOG.md

```diff
@@ -1,16 +1,16 @@
 # Changelog
 
-### Dependencies
+## [2.0.6](https://github.com/Financial-Times/dotcom-tool-kit/compare/create-v2.0.5...create-v2.0.6) (2022-07-12)
 
-* The following workspace dependencies were updated
-  * dependencies
-    * dotcom-tool-kit bumped from ^2.0.0 to ^2.1.0
 
 ### Dependencies
 
 * The following workspace dependencies were updated
   * dependencies
-    * dotcom-tool-kit bumped from ^2.2.0 to ^2.2.1
+    * dotcom-tool-kit bumped from ^2.2.2 to ^2.2.3
+
+## [2.0.5](https://github.com/Financial-Times/dotcom-tool-kit/compare/create-v2.0.4...create-v2.0.5) (2022-06-21)
```

Comment:
question: should this be dated (2022-06-20) as well because that's the date of when release please added the dependencies section https://github.com/Financial-Times/dotcom-tool-kit/blob/4f3f9b422b0771b8cdf3b7664e3b4dbc38d3dbe9/core/create/CHANGELOG.md?